src/js/components/mapWidgets/widgetContent/searchContent.tsx

```diff
@@ -18,9 +18,13 @@ const SearchContent: FunctionComponent = () => {
   const [formError, setFormError] = useState(false);
   const [latitudeInput, setLatitude] = useState('');
   const [longitudeInput, setLongitude] = useState('');
-  const selectedLanguage = useSelector(
-    (state: RootState) => state.appState.selectedLanguage
+  const { selectedLanguage, selectedSearchWidgetLayer } = useSelector(
+    (state: RootState) => state.appState
```

Comment:
I have been going through the application and fixing our selectors. Basically this way that we have been extracting properties, will always cause a re-render of the component. We need to extract values separately, or use something like `reselect` lib.

```suggestion
  const selectedLanguage = useSelector(
    (state: RootState) => state.appState.selectedLanguage
  const selectedSearchWidgetLayer = useSelector(
    (state: RootState) => state.appState.selectedSearchWidgetLayer
```

Reply:
done and done